src/trixi.h

```diff
@@ -0,0 +1,12 @@
+#ifndef LIBTRIXI_H_
```

Comment:
```suggestion
#ifndef TRIXI_H_
```
To match filename (needs to be changed below as well)